jiratools/error_logger.py

```diff
@@ -1,20 +1,35 @@
 """Error logging for jiratools."""
+from typing import List, Iterable
+
 from jiratools import add_comment, format_as_code_block
-from jiratools.formatting import format_autoupdate_jira_msg, format_as_jira_table
+from jiratools.formatting import (
+    DataArray,
+    format_autoupdate_jira_msg,
+    format_as_jira_table,
+)
+
+DataHeaders = List[str]
+
 
+class JiraEntry:
+    """Typing mock for Jira Entry data."""
 
-def add_jira_error_comment(jira_id, error_msg, **format_kwargs):
+    jira_id: str
+    error_message: str
```

Comment:
Did you intend for these two lines (17 and 18) to be part of the doc string? Is this some sort of static attribute typing in Python that I don't recognize (i would have expected an equal sign for something like that anyway)?

Reply:
So the diff is a little messy, but those are attributes on `class JiraEntry`, which is used places like [here](https://github.com/jolly-good-toolbelt/jiratools/pull/12/files#diff-1e1ab18b216d2d1bb58e4d5de7939f4cR77) to spec out an expected type. It does, incidentally, show up in the docs’ source code at least — https://bradsbrown.github.io/jiratools/_modules/jiratools/error_logger.html#JiraEntry

Reply:
👍